packages/cadl-ranch-specs/http/type/model/error/main.tsp

```diff
@@ -0,0 +1,104 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates usage of error models.")
+@scenarioService("/type/model/error")
+namespace Type.Model.Error;
+
+@doc("User model")
+model User {
+  username: string;
+  password: string;
```

Comment:
nit, do we really need to put a `password` property in test? It is not very probable, but it could trick some CredScan.